src/App.vue

```diff
@@ -62,8 +64,8 @@ export default {
     };
   },
   computed: {
-    isXS() {
-      return this.$vuetify.breakpoint.xsOnly;
+    isSM() {
+      return this.$vuetify.breakpoint.smAndDown;
```

Comment:
A função é `isSm` mas o breakpoint é `small and down`

Reply:
Verdade, nessa tu me pegou.